renderer/BUILD.gn

```diff
@@ -18,6 +18,7 @@ source_set("renderer") {
   deps = [
     "//base",
     "//brave/common",
+    "//chrome/common",
```

Comment:
What is this one needed for? Not seeing anything directly including "chrome/common".

Reply:
#include "chrome/common/plugin.mojom.h" in ChromeContentRendererClient. We can't have a `//chrome/renderer` dependency here because it would be a circular dependency

Reply:
I got a build error because the header wasn't created yet